src/components/reviews/Reviews.js

```diff
@@ -50,7 +50,7 @@ const Reviews = (props) => {
 
   const fetchCurrentReviews = () => {
     changeReviewsLoading(true);
-    apiController.getReviews(productId, { count: 100, sort: sort, pages: nextReviewsPage }).then((results) => {
+    apiController.getReviews(productId, { count: 100 * nextReviewsPage, sort: sort, pages: 1 }).then((results) => {
```

Comment:
interesting, I haven't thought about that

Reply:
Yeah, I had the thought that it'd be necessary because the plan was to fetch the reviews again when I sort them, but now that I'm thinking about it, it seems unecessary, because the only sort I can't do by hand is the 'relevance' sort, so if I just fetch them sorting by relevance, I can create the other two sorted arrays in React, which should save some bandwidth on fetching.